pootle/apps/pootle_store/models.py

```diff
@@ -40,7 +40,8 @@
 from pootle.core.models import Revision
 from pootle.core.storage import PootleFileSystemStorage
 from pootle.core.search import SearchBroker
-from pootle.core.url_helpers import get_editor_filter, split_pootle_path
+from pootle.core.url_helpers import (
+    get_editor_filter, split_pootle_path, PathResolver)
```

Comment:
Nit: Use styleguide recommendations on how to wrap long lines: "If you break in a statement with parentheses or braces, align to the braces" in http://docs.translatehouse.org/projects/translate-toolkit/en/latest/developers/styleguide.html#continuing-long-statements